extra/resources/ClusterMon.in

```diff
@@ -61,7 +61,7 @@ as seconds.
 <content type="integer" default="15000" />
 </parameter>
 
-<parameter name="extra_options">
+<parameter name="extra_options" reloadable="1">
```

Comment:
Also need to support reload-agent action (which can be no-op)

I think update and htmlfile would be reloadable too, they just set an argument when crm_mon is called. If stop removed htmlfile it would be different, but the htmlfile is left to show the last known status.